src/Data/LLVM/BitCode/GetBits.hs

```diff
@@ -90,24 +89,25 @@ instance MonadPlus GetBits where
 -- files get padded to a 32-bit boundary.
 data SubWord
```

Comment:
Now that there's only one constructor, it might be worth making `SubWord` a `newtype`, since it's probably quite central in the performance story.